rosetta-source/src/main/resources/cdm-sample-files/fpml-5-13/products/commodity-derivatives/expectations.json

```diff
@@ -5,11 +5,11 @@
 {"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex34-gas-put-option-european-floating-strike.xml","excludedPaths":-1,"externalPaths":86,"outstandingMappings":32,"validationFailures":10,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex35-call-option-gas-power-heat-rate-daily.xml","excludedPaths":-1,"externalPaths":77,"outstandingMappings":27,"validationFailures":5,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
 {"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex36-gas-call-option-european-spread-negative-premium-floating-strike.xml","excludedPaths":-1,"externalPaths":79,"outstandingMappings":25,"validationFailures":9,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"Commodity_Option","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex37-gold-forward-offered-rate.xml","excludedPaths":-1,"externalPaths":97,"outstandingMappings":1,"validationFailures":1,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_Basis","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-13/products/commodity-derivatives/com-ex37-gold-forward-offered-rate.xml","excludedPaths":-1,"externalPaths":97,"outstandingMappings":1,"validationFailures":2,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"InterestRate_IRSwap_Basis","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
```

Comment:
Validation errors due to use of commodity floating rate index which does not match any FloatingRateIndexEnum values